com.zeldaclassic.Zquest.yml

```diff
@@ -0,0 +1,65 @@
+app-id: com.zeldaclassic.Zquest
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --device=all
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+cleanup:
+  - /include
+  - "*.a"
+  - /lib/cmake
+  - /lib/debug
+  - /lib/libpng
+  - /lib/pkgconfig
+command: zlauncher
+rename-appdata-file: zquest.appdata.xml
+rename-desktop-file: zquest.desktop
+rename-icon: zquest
+modules:
+  - name: zquest
+    buildsystem: cmake-ninja
+    build-options:
+        build-args:
+          - --share=network
+    post-install:
+      - install -Dm644 zquest.appdata.xml -t ${FLATPAK_DEST}/share/metainfo/
+      - install -Dm644 zquest.desktop -t ${FLATPAK_DEST}/share/applications/
+      - icon_in="resources/zc.png";
+        icon_out="zquest.png";
+        for s in {32,64,128,192,256,512}; do
+        convert "${icon_in}" -resize "${s}" "${icon_out}";
+        install -p -Dm644 "${icon_out}" -t "${FLATPAK_DEST}/share/icons/hicolor/${s}x${s}/apps/";
+        done;
+      - find -L . -perm /a+x -type f -maxdepth 1 -name "*.so*" -exec cp -a {} ${FLATPAK_DEST}/lib/ \;
+      - find -L . -perm /a+x -type f -maxdepth 1 -not -name "*.so*" -exec cp -a {} ${FLATPAK_DEST}/bin/ \;
+      - cp -R RelWithDebInfo/* ${XDG_DATA_HOME}/
```

Comment:
This one doesn't fit yet. How can I make it so that the files end up in the user's data folder during installation?